modules/ROOT/pages/access-logging.adoc

```diff
@@ -17,22 +17,22 @@ If you do not specify attributes, the defaults are used. To see a list of the de
 
 == HTTP access log settings
 
-* Using a common log
-+
-To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:
+You can configure HTTP access log settings either for multiple endpoints that share common log settings or for individual endpoints.
 
+Settings for a common log::
+To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:
```

Comment:
Re-phrased the sentence as Acrolinx picked up this for being too long. Also, removed : and replaced with a period.